frontend/src/app/diff-versions/diff-versions.component.html

```diff
@@ -106,14 +106,21 @@ <h5>Endpoint changes</h5>
           </ngb-tab>
         </ngb-tabset>
       </div>
-      <div *ngIf="!this.validRangeSelected">
-        <p>
-          Choose two different versions from the table on the left using the radio buttons
-        </p>
+      <div *ngIf="searchComplete ? this.service.specifications.length >= 2 : false">
```

Comment:
Can't you specify that using `&&` rather than with a nested if? ;)

Reply:
It would make the conditional expressions really long, no? Is it necessarily better?